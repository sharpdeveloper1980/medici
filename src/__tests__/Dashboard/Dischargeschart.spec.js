import React, { Component } from "react";
import { create } from "react-test-renderer";
import Chart from 'react-apexcharts';
const Dischargeschart = (props) => {
    const apexDonutOpts = {
        chart: {
            height: 340,
            type: 'donut',
        },
        colors: ['#1164a0','#fea939','#1a716a'],
        legend: {
            show: false,
        },
        responsive: [
            {
                breakpoint: 376,
                options: {
                    chart: {
                        width: 250,
                        height: 250,
                    },
                    legend: {
                        position: 'bottom',
                    },
                },
            },
        ],
    };
    var apexDonutData = [0, 0, 0];
    return (
            <div>
                <div className="row">
                    <div className="col-sm-9">
                        <h6 style={{ fontSize:'12px' }} className="header-title mb-2 mt-1">Discharges</h6>
                    </div>
                    <div className="col-sm-3">
                        <select className="form-control form-control-sm" style={{ fontSize:'10px',height:'23px' }}>
                            <option>One Week</option>
                            <option>Two Week</option>
                        </select>
                    </div>
                </div>
                <div className="row">
                    <div className="col-sm-6">
                        <Chart options={apexDonutOpts} series={apexDonutData} type="pie" height={200} className="apex-charts" />
                    </div>
                    <div className="col-sm-6 chart_data_info">
                        <div className="chart-widget-list">
                            <p>
                                <i className="mdi mdi-circle discharges_commenced"></i> Discharges Not Commenced
                                <span className="float-right">0</span>
                            </p>
                            <p>
                                <i className="mdi mdi-circle discharges_process"></i> Discharges In Process
                                <span className="float-right">0</span>
                            </p>
                            <p>
                                <i className="mdi mdi-circle discharges_completed"></i> Discharges Completed
                                <span className="float-right">0</span>
                            </p>
                        </div>
                    </div>
                </div>
                
               
            </div>
    );
}

describe("Dischargeschart component", () => {
  test("Matches the snapshot", () => {
    const dischargeschart = create(<Dischargeschart />);
    expect(dischargeschart.toJSON()).toMatchSnapshot();
  });
});