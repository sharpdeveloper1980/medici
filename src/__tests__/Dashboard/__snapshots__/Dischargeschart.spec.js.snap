// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dischargeschart component Matches the snapshot 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-sm-9"
    >
      <h6
        className="header-title mb-2 mt-1"
        style={
          Object {
            "fontSize": "12px",
          }
        }
      >
        Discharges
      </h6>
    </div>
    <div
      className="col-sm-3"
    >
      <select
        className="form-control form-control-sm"
        style={
          Object {
            "fontSize": "10px",
            "height": "23px",
          }
        }
      >
        <option>
          One Week
        </option>
        <option>
          Two Week
        </option>
      </select>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-sm-6"
    >
      <div
        className="apex-charts"
        height={200}
        options={
          Object {
            "chart": Object {
              "height": 340,
              "type": "donut",
            },
            "colors": Array [
              "#1164a0",
              "#fea939",
              "#1a716a",
            ],
            "legend": Object {
              "show": false,
            },
            "responsive": Array [
              Object {
                "breakpoint": 376,
                "options": Object {
                  "chart": Object {
                    "height": 250,
                    "width": 250,
                  },
                  "legend": Object {
                    "position": "bottom",
                  },
                },
              },
            ],
          }
        }
        series={
          Array [
            0,
            0,
            0,
          ]
        }
        type="pie"
        width="100%"
      />
    </div>
    <div
      className="col-sm-6 chart_data_info"
    >
      <div
        className="chart-widget-list"
      >
        <p>
          <i
            className="mdi mdi-circle discharges_commenced"
          />
           Discharges Not Commenced
          <span
            className="float-right"
          >
            0
          </span>
        </p>
        <p>
          <i
            className="mdi mdi-circle discharges_process"
          />
           Discharges In Process
          <span
            className="float-right"
          >
            0
          </span>
        </p>
        <p>
          <i
            className="mdi mdi-circle discharges_completed"
          />
           Discharges Completed
          <span
            className="float-right"
          >
            0
          </span>
        </p>
      </div>
    </div>
  </div>
</div>
`;
